flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/sql/SortITCase.scala

```diff
@@ -55,14 +55,8 @@ class SortITCase(
     tEnv.registerDataSet("MyTable", ds)
 
     val expected = sortExpectedly(tupleDataSetStrings)
-    val results = tEnv.sql(sqlQuery).toDataSet[Row].mapPartition(rows => Seq(rows.toSeq)).collect()
-
-    val result = results
-      .filterNot(_.isEmpty)
-      .sortBy(_.head)(Ordering.by(f=> f.toString))
```

Comment:
The problem here is the string conversion which results in a lexicographical order ("21,....." sorts before "5,...").
We should change this to `.sortBy(_.head)(Ordering.by(r => (r.getField(0), r.getField(1))))`